python/orca/example/learn/pytorch/brainMRI/brainMRI.py

```diff
@@ -12,61 +12,44 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+# ==============================================================================
+# Most of the pytorch code is adapted from S0MNATHS's notebook for
+# BrainMRI dataset.
+# https://www.kaggle.com/s0mnaths/brain-mri-unet-pytorch/notebook
 #
 
 from __future__ import print_function
-from UNET import UNet
-from dataset import *
-
-import os
-from os.path import exists
-from os import makedirs
-import argparse
+from torchvision.utils import make_grid
```

Comment:
make the import cleaner; i.e. group torch related imports together, bigdl related together